.circleci/config.yml

```diff
@@ -39,7 +39,8 @@ integration_tests: &integration_tests
     # v1-docker,
     integration-test-skip-install,
     integration-test-install-v1,
-    replace-pip-aws-test
+    replace-pip-aws-test,
+    use-role-arn
```

Comment:
Has to be added to the relevant workflow.

Reply:
should be in.